api/client.php

```diff
@@ -290,7 +291,7 @@ function register() {
 
 	function seemain() {
 		echo '<capu><info><code>-1</code>';
-		$con=mysql_connect("localhost","root","19951025");
+		@$con=mysql_connect(CAPU_DB_HOST, CAPU_DB_USER, CAPU_DB_PWD);
```

Comment:
Yeah, I really appreciate this change. That's just what I want to do when I propose issue #13.
But there is also another concern, whether we can only use ONE `$con` instance all over the code base?
And for `mysql_connect`, it is deprecated, let's change it in the future.
We can use `MySQLi` extension [1] with its contruction method [2].
[1] http://php.net/manual/en/class.mysqli.php
[2] http://php.net/manual/en/mysqli.construct.php